prompt_poet/template_loaders.py

```diff
@@ -24,7 +24,7 @@ def id(self) -> str:
         pass
 
 
-def _parse_template_path(template_path: str) -> tuple[str, str]:
+def parse_template_path(template_path: str) -> tuple[str, str]:
```

Comment:
nit: this should either be in a commonly imported module or you should move the gcs_dict_template_loader.py implementation into this template_loaders.py...why have LocalFSTemplateLoader implementation here and the GCS one in its own module? doesn't make sense